todolist/app/components/Footer/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import { StyleSheet, View, TouchableOpacity, Text } from 'react-native'
+import { connect } from 'react-redux';
+
+import { actionCreators } from '../../redux/Todo/actions';
+
+const styles = StyleSheet.create({
+  footer: {
+    backgroundColor: 'white',
```

Comment:
create a constant folder at the same level that redux. Then inside create a colors.js file. You can extract all colors there. Try to use hex to define them... except for translucid colors (use rgba in that case)